stable/oauth2-proxy/README.md

```diff
@@ -22,6 +22,16 @@ $ helm install stable/oauth2-proxy --name my-release
 
 The command deploys oauth2-proxy on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
+## Installing on openshift
+
```

Comment:
this could show the values.yaml changes needed to make it work on openshift.  maybe we could add a `openshift-values.yaml` in the base dir to show a full example ?